cpp/modmesh/python/common.hpp

```diff
@@ -285,15 +285,15 @@ WrapBase
     wrapper_type & METHOD(Args &&... args)                                    \
     {                                                                         \
         m_cls.METHOD(std::forward<Args>(args)...);                            \
-        return *static_cast<wrapper_type *>(this);                            \
+        return *static_cast<std::add_pointer_t<wrapper_type>>(this);          \
```

Comment:
use modern C++

Reply:
If this does nothing to the resolution of the bad assembly, please avoid the refactoring change in this **changeset**.

You may use another refactoring PR.

Reply:
sure